PubNub/Core/PubNub+Publish.m

```diff
@@ -413,6 +465,10 @@ - (PNRequestParameters *)requestParametersForMessage:(NSString *)message toChann
         [parameters addQueryParameter:[PNString percentEscapedString:metadata] forFieldName:@"meta"];
     }
     
+    if (!shouldReplicate) {
+        [parameters addQueryParameter:@"1" forFieldName:@"no-rep"];
```

Comment:
i just double checked the docs and it's `norep`, I made a mistake in the pivotal here is the same in [java](https://github.com/pubnub/java/blob/768b539a728abfb4e273c29c3fcb31260aa8f859/src/main/java/com/pubnub/api/endpoints/pubsub/Publish.java#L96)